test/unit/test_disk_tensor.py

```diff
@@ -31,6 +31,7 @@ def test_load_convnext(self): compare_weights_both('https://dl.fbaipublicfiles.c
   # CUDACPU architecture is sm_35 but we need at least sm_70 to run fp16 ALUs
   @unittest.skipIf(Device.DEFAULT in ["GPU", "LLVM", "CUDA"] and CI, "fp16 broken in some backends")
   @unittest.skipIf(Device.DEFAULT == "TORCH", "torch doesn't support the way we load bfloat (cast to uint32)")
+  @unittest.skipIf(Device.DEFAULT == "WEBGPU", "float16 and uint16 are broken in webgpu")
```

Comment:
can this use `is_dtype_supported` too? can make `is_dtype_supported` take multiple dtypes then it's clear what are the types that need to support for each test